test/Number.js

```diff
@@ -0,0 +1,23 @@
+// Prepare chai
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Number', function() {
+  describe('static isNumber', function() {
+    it('string is not a number', function() {
+      expect(Number.isNumber('a')).to.be.false;
+    });
+
+    it('null is not a number', function() {
+      expect(Number.isNumber(null)).to.be.false;
+    });
+
+    it('undefined is not a number', function() {
+      expect(Number.isNumber('undefined')).to.be.false;
+    });
+
+    it('1 is a number', function() {
+      expect(Number.isNumber(1)).to.be.true;
+    });
+  });
+});
```

Comment:
Each text file should end with new line. Please read more about it here: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Thanks for the info, I added a new line to end of file